Common Programming Concepts/Data Types/Arrays/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+pub fn arr() -> &'static str {
+    let a = ["All work and no play makes Jack a dull boy"; 100];
+
+    if a.len() >= 100 {
```

Comment:
I havve an inspection in Idea here to extract the return from the if clause. Let's apply it.
```
return if a.len() >= 100 {
        "Wow, that's a big array!"
    } else {
        "Meh, I eat arrays like that for breakfast."
    }
```